### ISSUE TITLE
dropdown module: works with html select tag; but how about select with rows attribute.

### ISSUE THREAD

**Issue by rnealxp** [Semantic-Org/Semantic-UI#6178]
--Sorry if this is in the wrong place. It is a feature request (and a strikingly missing one I think), but I could not figure how else to post (let me know).
--The dropdown module works well with html select tag. This helps people port code to Semantic-UI. What is missing is the use case where user needs to select from a dropdown or list that behaves like html select tag does. Yes, Semantic-UI has lists, but these make it overly complex to let the user make a choice. If you would just tweak the behavior of styling for dropdown when an html select tag has the size attribute set. Recall, when the size attribute is set, multiple items in the dropdown are visible to the user.

**Comment by milahu** [comment]
yes, we need this for backward compatibility, to have useable forms with javascript off.

just insert
`select.ui.dropdown[size]{height:auto}
`
after
`select.ui.dropdown{height:38px`
